src/ui/windows/TogglDesktop/TogglDesktop/Program.cs

```diff
@@ -45,7 +45,10 @@ private static void OnBeforeStartup()
     public static void Shutdown(int exitCode)
     {
         Toggl.Clear();
-        Environment.Exit(exitCode);
+        if (Environment.OSVersion.Version.Major == 6 && Environment.OSVersion.Version.Minor == 1)

```

Comment:
It is unclear what exactly this means by just reading it (even though I understand this and could look up to which Windows version this corresponds to).
I'd extract an extension or utility method (smth like `IsWindows7()`) + maybe add a small comment about this issue.

Reply:
Good point, thanks.
Fixed now.